### DISCUSSION TITLE
CUDA out of memory after re-install - missing something ciritcal! :(

### DISCUSSION THREAD

**Discussion by telemole** [vladmandic/sdnext#848] [Q&A]
Hi folks, I had this distribution purring along... and i deleted some old directories, turned out to the version of python i was pointing at - i reinstalled python, but not my install of VLAD crashes with memory errors over 512X512 - i was doing 1024x1024 no problem before, so i know my 3080 TI can do it... 

TL/DR : Can anyone tell me how to resolve this error? would love to get back to where i was :( thank you!!
RuntimeError: CUDA out of memory. Tried to allocate 8.00 GiB (GPU 0; 12.00 GiB total capacity; 10.18 GiB already
allocated; 0 bytes free; 10.21 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting
max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

**Comment by vladmandic** [comment]
it has 10gb already allocated and its asking for 8gb more - no way thats just 512x512, make sure you don't have hires or upscale settings set somewhere.
you can run with `webui --debug` to trace whats going on.

**Reply by telemole** [reply]
right you are! NO clue what was wrong but it was running away with GPU ram - re-installed EVERYTHING and ran debug to watch it all,  and I'm up and running!

**Reply by Tillerz** [reply]
Where we are at resolutions... when you are in img2img and set a resolution for the output... and then later start up the whole thing again to do some txt2img, it takes over the resolution fomr img2img, but it keeps the scale factor.

So if you had
- txt2img 512x512 and scale 2 for 1024x1024
- switch to img2img, set resolution to 1024x1024
- quit doing things, later start the server to do things ...
- now you suddenly have txt2img 1024x0124 and scale 2 for 2048x2028 which breaks everywhere except you have some monster like an RTX 4090

Is there a way to prevent that?
